projects/ngx-interactive-paycard-demo/src/assets/brand-icons/diners.svg

```diff
@@ -0,0 +1,13 @@
+<svg width="750" height="471" viewBox="0 0 750 471" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
I change the logos for generic logos

Reply:
Okok thank you! I hope I'll find the editable version of my original generic company logo. I could update that logo to `Company1`, `Company2` etc. and then it would fit the design. But for now, these generic icons are ok, I'll take care of this when the PR will be merged.